src/main/webapp/js/instructorFeedbackResults.js

```diff
@@ -209,14 +209,21 @@ function toggleCollapse(e, pans) {
     var panels = pans || $('div.panel-collapse');
 
     if ($(e).html().trim().startsWith(expand)) {
-        isExpandingAll = true;
-        var i = 0;
-        for (var idx = 0; idx < panels.length; idx++) {
-            if ($(panels[idx]).attr('class').indexOf('in') === -1) {
-                setTimeout(showSingleCollapse, 50 * i, panels[idx]);
-                i++;
+        $('.ajax-response-auto').click();
```

Comment:
I'd rather not merge this until this is fixed properly. The current solution seems is too hackish. We shouldn't hack-first-fix-later unless it is an urgent issue, which this is not. :-p